ingress-nginx-controller-1.12.yaml

```diff
@@ -498,6 +502,67 @@ pipeline:
         && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/usr/bin/nginx
 
 subpackages:
+  - name: ingress-nginx-controller-bitnami-compat-${{vars.nginx-ingress-major-minor}}
+    description: Compatibility package with bitnami/nginx-ingress-controller image
+    dependencies:
+      provides:
+        - ingress-nginx-controller-bitnami-compat=${{package.full-version}}
+      runtime:
```

Comment:
Just double checking - do we need to add other GNU tools here? https://eng.inky.wtf/docs/images/package-and-image-guides/bitnami/#runtime-dependencies (See the scripts below)

Reply:
@Dentrax do you mean `coreutils` or `net-tools`?

Reply:
I've just added coreutils BTW, thanks @Dentrax

Reply:
Just double-checking this again, do we have all deps as provided here? https://github.com/bitnami/containers/blob/86622352cc33c05d23f6e97722c1364ca38ceeeb/bitnami/nginx-ingress-controller/1/debian-12/Dockerfile#L29

Reply:
They all come from there and checking from `apt` @Dentrax, except `libhogweed6`  as noted below.